CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ option(BUILD_COVER "Generate test coverage" OFF)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
 
 include(ClangFormat)
+if (WIN32)
+        set(CMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP ON)
+    include(InstallRequiredSystemLibraries)
+endif()
```

Comment:
This install MSVC(P/R).dll into the resulting directory, it depends what you want, provide the dlls or ask users to install the C++ redistributable?